ui/src/keyboardLayouts/sl_SI.ts

```diff
@@ -0,0 +1,157 @@
+  Z: { key: "KeyY", shift: true },
+  "Ž": { key: "Backslash", shift: true },
+  a: { key: "KeyA" },
+  b: { key: "KeyB" },
+  c: { key: "KeyC" },
+  "č": { key: "Semicolon"},
+  "ć": { key: "Quote"},
+  d: { key: "KeyD" },
+  "đ": { key: "BracketRight"},
+  e: { key: "KeyE" },
+  f: { key: "KeyF" },
+  g: { key: "KeyG" },
+  h: { key: "KeyH" },
+  i: { key: "KeyI" },
+  j: { key: "KeyJ" },
+  k: { key: "KeyK" },
+  l: { key: "KeyL" },
+  m: { key: "KeyM" },
+  n: { key: "KeyN" },
+  o: { key: "KeyO" },
+  p: { key: "KeyP" },
+  q: { key: "KeyQ" },
+  r: { key: "KeyR" },
+  s: { key: "KeyS" },
+  "š": { key: "BracketLeft"},
+  t: { key: "KeyT" },
+  u: { key: "KeyU" },
+  v: { key: "KeyV" },
+  w: { key: "KeyW" },
+  x: { key: "KeyX" },
+  y: { key: "KeyZ" },
+  z: { key: "KeyY" },
+  "ž": { key: "Backslash"},
+  1: { key: "Digit1" },
+  "!": { key: "Digit1", shift: true },
+  2: { key: "Digit2" },
+  "\"": { key: "Digit2", shift: true },
+  3: { key: "Digit3" },
+  "#": { key: "Digit3", shift: true },
+  4: { key: "Digit4" },
+  "$": { key: "Digit4", shift: true },
+  5: { key: "Digit5" },
+  "%": { key: "Digit5", shift: true },
+  6: { key: "Digit6" },
+  "&": { key: "Digit6", shift: true },
+  7: { key: "Digit7" },
+  "/": { key: "Digit7", shift: true },
+  8: { key: "Digit8" },
+  "(": { key: "Digit8", shift: true },
+  9: { key: "Digit9" },
+  ")": { key: "Digit9", shift: true },
+  0: { key: "Digit0" },
+  "=": { key: "Digit0", shift: true },
+  "'": { key: "Minus" },
+  "?": { key: "Minus", shift: true },
+  "+": { key: "Equal" },
+  "*": { key: "Equal", shift: true },
+
+  "<": { key: "IntlBackslash" },
+  ">": { key: "IntlBackslash", shift: true },
+  ",": { key: "Comma" },
+  ";": { key: "Comma", shift: true },
+  ".": { key: "Period" },
+  ":": { key: "Period", shift: true },
+  "-": { key: "Slash" },
+  "_": { key: "Slash", shift: true },
+
+  "~": { key: "Digit1", shift: true },
+  "ˇ": { key: "Digit2", shift: true },
+  "^": { key: "Digit3", shift: true },
+  "˘": { key: "Digit4", shift: true },
+  "°": { key: "Digit5", shift: true },
+  "˛": { key: "Digit6", shift: true },
+  "`": { key: "Digit7", shift: true },
+  "˙": { key: "Digit8", shift: true },
+  "´": { key: "Digit9", shift: true },
+  "˝": { key: "Digit0", shift: true },
+  "¨": { key: "Minus", shift: true },
+  "¸": { key: "Equal", shift: true },
+  "\\": { key: "KeyQ", AltGr: true },
+  "|": { key: "KeyW", AltGr: true },
+  "€": { key: "KeyE", AltGr: true },
+  "÷": { key: "BracketLeft", AltGr: true },
+  "×": { key: "BracketRight", AltGr: true },
+  "[": { key: "KeyF", AltGr: true },
+  "]": { key: "KeyG", AltGr: true },
+  "ł": { key: "KeyK", AltGr: true },
+  "Ł": { key: "KeyL", AltGr: true },
+  "ß": { key: "Quote", AltGr: true },
+  "¤": { key: "Backslash", AltGr: true },
+  "@": { key: "KeyV", AltGr: true },
+  "{": { key: "KeyB", AltGr: true },
+  "}": { key: "KeyN", AltGr: true },
+  "§": { key: "KeyM", AltGr: true },
+  // "<": { key: "Comma", AltGr: true },
```

Comment:
Should these actually be commented-out?

Reply:
Please leave those in there, with a note, so I can take that situation into account as I do the keyboard layout work next on my schedule.